BTCPayServer/wwwroot/js/vaultbridge.ui.js

```diff
@@ -32,6 +32,7 @@ var vaultui = (function () {
         noWebsockets: new VaultFeedback("failed", "Web sockets are not supported by the browser.", "vault-feedback1", "no-websocket"),
         errorWebsockets: new VaultFeedback("failed", "Error of the websocket while connecting to the backend.", "vault-feedback1", "error-websocket"),
         bridgeConnected: new VaultFeedback("ok", "BTCPayServer successfully connected to the vault.", "vault-feedback1", "bridge-connected"),
+        vaultNeedUpdate: new VaultFeedback("failed", "BTCPayServer Vault is running, but you need to update. Download the latest version on <a target=\"_blank\" href=\"https://github.com/btcpayserver/BTCPayServer.Vault/releases/latest\">Github</a>.", "vault-feedback2", "vault-outdated"),
```

Comment:
```suggestion
        vaultNeedUpdate: new VaultFeedback("failed", "Your BTCPay Server Vault version is obsolete. Please <a target=\"_blank\" href=\"https://github.com/btcpayserver/BTCPayServer.Vault/releases/latest\">download</a> the latest version.", "vault-feedback2", "vault-outdated"),
```

Wording suggestion.
Instead of `obsolete`, can also be `outdated`.

Reply:
Obsolete is not the right word I prefer `outdated`.